### PR TITLE
Pashto language localization

### PR DESCRIPTION
**Pull request by isshort**
Added Pashto language localization.

### CODE REVIEW

**Comment by isshort** [comment]
is exist there  any issues?

**Comment by larryaasen** [comment]
I appreciate you adding the Pashto language localization.

In your code, the ps language should appear right after the nl language to keep them alphabetized.

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +17,7 @@ stream is updated each time the app resumes from the background.
 - Added support for checking for updates every time the app resumes from the background. (#272)
 - Changed the way Upgrader is initialized to support a stream of evaluation requests. The
 stream is updated each time the app resumes from the background.
+
```
Comment (larryaasen) [inline CHANGELOG.md:20]: What is this change for?

File: CHANGELOG.md [hunk 2]
```diff
@@ -86,7 +90,7 @@ stream is updated each time the app resumes from the background.
 ## 5.1.0
 
 - Updated the initialization of upgrader to use a future that will allow the initialize method to be called multiple times. This
-does not require any changes by app code.
+  does not require any changes by app code.
```
Comment (larryaasen) [inline CHANGELOG.md:93]: What is this change for?

File: CHANGELOG.md [hunk 3]
```diff
@@ -96,7 +100,7 @@ does not require any changes by app code.
 ## 5.1.0-alpha.1
 
 - Updated the initialization of upgrader to use a future that will allow the initialize method to be called multiple times. This
-does not require any changes by app code.
+  does not require any changes by app code.
```
Comment (larryaasen) [inline CHANGELOG.md:103]: What is this change for?

File: CHANGELOG.md [hunk 4]
```diff
@@ -120,31 +124,31 @@ does not require any changes by app code.
 ## 4.11.0
 
 - Updated package dependencies:
-    - mockito to ">=5.0.0 <5.4.0"
+  - mockito to ">=5.0.0 <5.4.0"
```
Comment (larryaasen) [inline CHANGELOG.md:127]: What is this change for?

File: CHANGELOG.md [hunk 5]
```diff
@@ -120,31 +124,31 @@ does not require any changes by app code.
...
 
 - Updated minimum Flutter version to 2.5.0 and Dart to 2.14.0.
 - Updated package dependencies:
-    - device_info_plus to ">=3.2.0 <7.0.0"
-    - html to ">=0.15.0 <=0.15.1"
-    - http to ">=0.13.0 <=0.13.5"
-    - package_info_plus to ">=1.3.0 <3.0.0"
-    - shared_preferences to ">=2.0.3 <2.1.0"
-    - url_launcher to ">=6.1.0 <= 6.1.5"
+  - device_info_plus to ">=3.2.0 <7.0.0"
```
Comment (larryaasen) [inline CHANGELOG.md:135]: What is this change for?

File: README.md [hunk 6]
```diff
@@ -493,7 +518,9 @@ flutter: upgrader: showDialog title: Update App?
 flutter: upgrader: showDialog message: A new version of Upgrader is available! Version 5.81 is now available-you have 1.0.0.
 flutter: upgrader: showDialog releaseNotes: Thanks for using Google Maps! This release brings bug fixes that improve our product to help you discover new places and navigate to them.
 ```
+
 Also, please include the upgrader version number from the pubspec.lock file, which should look something like this:
+
```
Comment (larryaasen) [inline README.md:523]: What is this change for?

**Comment by larryaasen** [comment]
@isshort If you would like this PR to be merged, you first need to address the comments and the merge conflict.

**Comment by larryaasen** [comment]
@isshort Any update on this?

File: CHANGELOG.md (hunk 3 above)
Comment (isshort) [reply CHANGELOG.md:103]: This was my IDE's automatic changes.

File: CHANGELOG.md (hunk 5 above)
Comment (isshort) [reply CHANGELOG.md:135]: This was my IDE's automatic changes.

File: README.md (hunk 6 above)
Comment (isshort) [reply README.md:523]: This was my IDE's automatic changes.

**Review by mzisshort — APPROVED** [review]

**Comment by larryaasen** [comment]
@isshort I can't merge this PR until you address the comments in this PR.

**Comment by larryaasen** [comment]
@isshort I cannot merge this PR unless the random file changes are removed.

**Comment by isshort** [comment]
> @isshort I cannot merge this PR unless the random file changes are removed.

Okay I'll fix it asap.

**Comment by larryaasen** [comment]
Closed due to inactivity.

**Pull request closed without merging**